examples/green/Dockerfile

```diff
@@ -87,15 +80,35 @@ RUN echo "PermitRootLogin yes" > /etc/ssh/sshd_config.d/rootlogin.conf
 # Add default snapshotter setup
 ADD snapshotter.yaml /etc/elemental/config.d/snapshotter.yaml
 
+# Add specific Grub bootargs for RISC-V
+ADD riscv_bootargs.cfg /tmp/riscv_bootargs.cfg
+RUN ARCH=$(uname -m); \
+    [[ "${ARCH}" == "riscv64" ]] && mv -f /tmp/riscv_bootargs.cfg /etc/elemental/bootargs.cfg || true
+
 # Generate initrd with required elemental services
-RUN elemental --debug init --force
+RUN ARCH=$(uname -m); \
+    if [[ "${ARCH}" != "riscv64" ]]; then \
+      ADD_FEATURES+=" grub-default-bootargs"; \
+      [[ "${ARCH}" == "aarch64" ]] && ADD_FEATURES+=" arm-firmware"; \
+    fi; \
+    elemental --debug init --force \
```

Comment:
I think we should keep the default features here since these are what's tested in the CI.

Reply:
What do you mean? The code will keep the defaults for x86 (yes explicitly written) and only change for arm64 (adding `arm-firmware`) and riscv64 because this one needs specific Grub options.

Reply:
Yes, but we want to test that the default features added through the implicit case works (also `autologin` is not one of the default features)

Reply:
In that case I will have to either use a dedicated Dockerfile or defined the list of features only for riscv64 (and arm64 maybe?) and keep the default for x86.

WDYT is the better way for you? I first tried to avoid a pretty much full copy/paste of the Dockerfile only for riscv64, but if you think that an easier way to do it
 that's fine.

Reply:
In that case I will have to either use a dedicated Dockerfile or defined the list of features only for riscv64 (and arm64 maybe?) and keep the default for x86.

WDYT is the better way for you? I first tried to avoid a pretty much full copy/paste of the Dockerfile only for riscv64, but if you think that an easier way to do it
 that's fine.

Reply:
Sure, I think using the defined features for riscv64/arm64 is fine, as long as we stick with the implicit configuration for x86! :+1:

Reply:
Should be OK, let me know if something else needs to be modified ;-)

Reply:
Looks good to me! Nice job! :+1: